src/NOTD.SC2Map/MapScript.galaxy

```diff
@@ -12343,9 +12367,9 @@ void gf_RestoreLeaverControl () {
     }
 }
 
-void gf_SetCombatKnifeDamage (int lp_player) {
+void gf_SetCombatKnifeDamage (int lp_triggeringPlayer) {
     // Implementation
-    CatalogFieldValueModify(c_gameCatalogEffect, "CombatKnifeDamage", "Amount", lp_player, FixedToString(((IntToFixed(gv_playerRank[lp_player]) - 1.0) * 0.5), c_fixedPrecisionAny), c_upgradeOperationAdd);
+    CatalogFieldValueModify(c_gameCatalogEffect, "CombatKnifeDamage", "Amount", lp_triggeringPlayer, FixedToString(((IntToFixed(gv_playerRank[lp_triggeringPlayer]) - 1.0) * 0.5), c_fixedPrecisionAny), c_upgradeOperationAdd);
 }
```

Comment:
I have no idea what's going on here. I never touched this function, and Galaxy Editor decided to change the parameter name (which does not affect the mechanics at all).